Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: build
+
+ifdef CI
+ENGINE := docker
```

Comment:
Just want to know the reason to choose the docker engine for CI here

Reply:
The workflow is running in an ubuntu based environment.  docker is the default container engine in Ubuntu.